src/PKSim.Presentation/Presenters/Simulations/PopulationPKAnalysisPresenter.cs

```diff
@@ -70,7 +70,7 @@ public void CalculatePKAnalyses(IPopulationDataCollector populationDataCollector
          _globalPKAnalysisPresenter.CalculatePKAnalysis(new[] { simulation });
 
          var pkParametersCache = extractPKParameters(populationAnalysis, simulation);
-         if (!pkParametersCache.Any())
+         if (!pkParametersCache.Any() || pkParametersCache.All(x => !x.Any()))
```

Comment:
That's the bug. The intent here is to make sure to stop if there's no parameters in the cache, but what's actually created is parameters without any values.

Reply:
It's a cache of QuantityPKParameters. So they are parameters that are empty of results.